tests/test-maximize.lua

```diff
@@ -82,10 +82,11 @@ local steps = {
         local bw      = c.border_width
 
         assert(c.fullscreen)
-        assert(new_geo.x-bw==sgeo.x)
-        assert(new_geo.y-bw==sgeo.y)
-        assert(new_geo.x+new_geo.width+bw==sgeo.x+sgeo.width)
-        assert(new_geo.y+new_geo.height+bw==sgeo.y+sgeo.height)
+
+        assert(new_geo.x==sgeo.x)
+        assert(new_geo.y==sgeo.y)
+        assert(new_geo.x+new_geo.width+2*bw==sgeo.x+sgeo.width)
+        assert(new_geo.y+new_geo.height+2*bw==sgeo.y+sgeo.height)
```

Comment:
Do I understand correctly that the code doing ` window:geometry(window.screen.geometry)` was actually wrong (since it ignored the border width) and the wrongness managed to end up in a test? Yuck.